credentials/alumni/credential.json

```diff
@@ -2,7 +2,7 @@
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
     "https://www.w3.org/2018/credentials/examples/v1",
-    "https://playground.chapi.io/examples/alumni/context-v1.json"
+    "https://contexts.vcplayground.org/examples/alumni-context-v1.json"
```

Comment:
NOTE: The contexts will be now served by https://github.com/credential-handler/chapi-playground-contexts-server/pull/1